(async () => {
  try {

    await require('./simulate2');
  } catch (err) {
    console.error(err);
  }

})();
